lib/DataObject/ClassesManager.php

```diff
@@ -0,0 +1,100 @@
+<?php
+/**
+ * Pimcore
+ *
+ * This source file is available under two different licenses:
+ * - GNU General Public License version 3 (GPLv3)
+ * - Pimcore Enterprise License (PEL)
+ * Full copyright and license information is available in
+ * LICENSE.md which is distributed with this source code.
+ *
+ * @copyright  Copyright (c) Pimcore GmbH (http://www.pimcore.org)
+ * @license    http://www.pimcore.org/license     GPLv3 and PEL
+ */
+
+namespace Pimcore\DataObject;
+
+use Pimcore\Db;
+use Pimcore\Db\Connection;
+use Pimcore\Model\DataObject\ClassDefinition;
+
+/**
+ * Class ClassesManager
+ * @package Pimcore\DataObject
+ */
+class ClassesManager
+{
+    public const SAVED = 'saved';
+    public const CREATED = 'created';
+    public const DELETED = 'deleted';
+
+    /**
+     * @var Db
+     */
+    private $db;
+
+    /**
+     * ClassesManager constructor.
+     * @param Connection $db
+     */
+    public function __construct(Connection $db)
+    {
+        $this->db = $db;
+    }
+
+    /**
+     * Delete all classes from db
+     */
+    public function deleteClasses(): array
```

Comment:
We should name this method accordingly to what it actually does. 
What about `cleanupDeletedClassDefinitions()` or anything more meaningful?